docs/google/floquet.ipynb

```diff
@@ -0,0 +1,1341 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "VJ3IC7PZN_F6"
+   },
+   "source": [
+    "## Details on the calibration tool"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "kY8lN-umLx6u"
+   },
+   "source": [
+    "In more detail, assuming we have a number-convserving two-qubit unitary gate, Floquet calibration (FC) returns fast, accurate estimates for the relevant angles to be calibrated. The `cirq.PhasedFSimGate` has five parameters $\\theta$, $\\zeta$, $\\chi$, $\\gamma$, $\\phi$ with unitary matrix\n",
+    "\n",
+    "$$\n",
+    "\\left[ \\begin{matrix}\n",
+    "1 & 0                                          & 0                                         & 0      \\\\\n",
+    "0 &    \\exp(-i \\gamma - i \\zeta) cos( \\theta ) & -i \\exp(-i \\gamma + i \\chi) sin( \\theta ) & 0 \\\\\n",
+    "0 & -i \\exp(-i \\gamma - i \\chi) sin( \\theta )  &    \\exp(-i \\gamma + i \\zeta) cos( \\theta) & 0    \\\\\n",
+    "0 & 0                                          & 0                               & \\exp(-2 i \\gamma -i \\phi )  \n",
+    "\\end{matrix} \\right]\n",
+    "$$\n",
+    "\n",
+    "With Floquet calibration, every angle but $\\chi$ can be calibrated. In experiments, we have found these angles change when gates are run in parallel. Because of this, we perform FC on entire moments of two-qubits gates and return different calibrated angles for each. \n",
+    "\n",
+    "After calibrating, one needs to adjust for these angles in the circuit to be implemented. The simplest correction is for single-qubit $Z$ rotations and works by adding $R_z$ gates before and after the two-qubit gates in question. For many circuits, even this simplest correction can lead to a significant improvement in results. We provide methods for doing this in this notebook and analyze results for an example circuit.\n",
+    "\n",
+    "We do not attempt to correct the misaligned iSWAP rotation or the additional two-qubit phase in this notebook. This is a non-trivial task and we do currently have simple tools to achieve this. It is up to the user to correct for these as best as possible."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "id": "-jOmHd7lLyDn"
+   },
+   "source": [
+    "*Note*: The Floquet calibration API and this documentation is ongoing work. The amount by which errors are reduced may vary from run to run and from circuit to circuit."
```

Comment:
For the note panel, no need for bolding: 

```suggestion
    "Note: The Floquet calibration API and this documentation is ongoing work. The amount by which errors are reduced may vary from run to run and from circuit to circuit."
```

Reply:
Removed here and elsewhere